scripts/ci/check-required-contexts.py

```
@@ -262,18 +375,302 @@ def apply(path: Path, mutate) -> dict | None:
+        code = _invoke_main_live(live_text)
+        if code != EXIT_MATCH:
+            print(
+                f"self-test: live {label} must exit {EXIT_MATCH} via main_live, got {code}",
+                file=sys.stderr,
+            )
+            return 1
+
+    drift_cases = [
+        (
+            "context_added",
+            lambda t: _baseline_live_json(expected + ["extra-context"], strict=expected_strict),
```

Ruff (0.16.1)

[warning] Consider `[*expected, "extra-context"]` instead of concatenation

Replace with `[*expected, "extra-context"]`

(RUF005)